app/mutations/application/createApplicationRevisionStatusEdgeMutation.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ConnectionHandler,
+  graphql,
+  DeclarativeMutationConfig
+} from 'react-relay';
+import RelayModernEnvironment from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  createApplicationRevisionStatusEdgeMutationVariables,
+  createApplicationRevisionStatusEdgeMutation as createApplicationRevisionStatusEdgeMutationType
+} from 'createApplicationRevisionStatusEdgeMutation.graphql';
+import BaseMutation from 'mutations/BaseMutation';
+
+const mutation = graphql`
+  mutation createApplicationRevisionStatusEdgeMutation(
+    $input: CreateApplicationRevisionStatusInput!
+    $connections: [ID!]!
+  ) {
+    createApplicationRevisionStatus(input: $input) {
+      applicationRevisionStatusEdge @appendEdge(connections: $connections) {
```

Comment:
Created this new mutation on an `Edge` because the status is now fetched from a connection - [see explanation below](https://github.com/bcgov/cas-ciip-portal/pull/1644/files#r623266796)